type.go

```diff
@@ -640,6 +730,48 @@ func (t floatType) AssignValue(dst reflect.Value, src Value) error {
 	switch dst.Kind() {
 	case reflect.Float32, reflect.Float64:
 		dst.SetFloat(float64(v))
+	case reflect.Pointer:
+		underlying := reflect.TypeOf(dst).Elem().Kind()
+		switch underlying {
+		case reflect.Float32:
+			dst.Set(reflect.ValueOf(&v))
+		case reflect.Float64:
+			val := float64(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Int8:
+			val := int8(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Int16:
+			val := int16(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Int32:
+			val := int32(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Int64:
+			val := int64(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Uint8:
+			val := uint8(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Uint16:
+			val := uint16(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Uint32:
+			val := uint32(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Uint64:
+			val := uint64(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Int:
+			val := int(v)
+			dst.Set(reflect.ValueOf(&val))
+		case reflect.Uint:
+			val := uint(v)
+			dst.Set(reflect.ValueOf(&val))
```

Comment:
Same here, let's not introduce type conversions from int to float that did not exist before.

Reply:
removed the conversion.